src/constants.tsx

```diff
@@ -1,20 +1,80 @@
 export const url = 'https://api.apiumando.info/'
 
 export const genres = [
```

Comment:
```suggestion
export const genres: Genre[] = [
```

You can explicitly type this.

Reply:
Done